### ISSUE TITLE
Update dependencies to fix security alert

### ISSUE THREAD

**Issue by jelmerdemaat** [developit/microbundle#473]
Hi there! Microbundle is a devDependency in one of my projects, and I get a security alert from Github (see screenshot below). It seems microbundle is depending on the `braces` package somewhere. Can this maybe be updated?

<img width="769" alt="Schermafbeelding 2019-09-04 om 23 45 08" src="https://user-images.githubusercontent.com/942560/64294126-6509bd00-cf6e-11e9-87f3-20e02a545e4e.png">

**Comment by KeironLowe** [comment]
+1

**Comment by piotr-cz** [comment]
Current output from `npm audit`

```

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Regular Expression Denial of Service

  Package         braces

  Patched in      >=2.3.1

  Dependency of   microbundle [dev]

  Path            microbundle > rollup-plugin-typescript2 > rollup-pluginutils
                  > micromatch > braces

  More info       https://npmjs.com/advisories/786


  Moderate        Denial of Service

  Package         js-yaml

  Patched in      >=3.13.0

  Dependency of   microbundle [dev]

  Path            microbundle > rollup-plugin-postcss > cssnano > postcss-svgo
                  > svgo > js-yaml

  More info       https://npmjs.com/advisories/788


  High            Code Injection

  Package         js-yaml

  Patched in      >=3.13.1

  Dependency of   microbundle [dev]

  Path            microbundle > rollup-plugin-postcss > cssnano > postcss-svgo
                  > svgo > js-yaml

  More info       https://npmjs.com/advisories/813


  Moderate        Cross-Site Scripting

  Package         serialize-javascript

  Patched in      >=2.1.1

  Dependency of   microbundle [dev]

  Path            microbundle > rollup-plugin-terser > serialize-javascript

  More info       https://npmjs.com/advisories/1426

found 4 vulnerabilities (1 low, 2 moderate, 1 high) in 2057 scanned packages
  4 vulnerabilities require manual review. See the full report for details.

```

**Comment by ForsakenHarmony** [comment]
This shoud be fixed by now
